2018/10/4 14:59:46 [main] *** Start to run FindCoin
2018/10/4 14:59:46 [main] *** Auth:tsc
2018/10/4 14:59:46 [main] *** Version:0.0.0.1
2018/10/4 14:59:46 [main] *** CreateDate:2018-07-25
2018/10/4 14:59:46 [main] *** LastModify:2018-08-08
2018/10/4 14:59:51 [block3] errinfo:
2018/10/4 14:59:51 [block3] System.Net.WebException
2018/10/4 14:59:51 [block3] No connection could be made because the target machine actively refused it No connection could be made because the target machine actively refused it
2018/10/4 14:59:51 [block3]    at System.Net.HttpWebRequest.GetResponse()
   at System.Net.WebClient.GetWebResponse(WebRequest request)
   at System.Net.WebClient.DownloadBits(WebRequest request, Stream writeStream)
   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
   at System.Net.WebClient.DownloadString(Uri address)
   at System.Net.WebClient.DownloadString(String address)
   at FindCoin.Block.FindBlock.getBlockHeightFromRpc() in C:\Users\tlan\NEO\FindCoin\FindCoin\Block\FindBlock.cs:line 71
   at FindCoin.Block.FindBlock.run() in C:\Users\tlan\NEO\FindCoin\FindCoin\Block\FindBlock.cs:line 53
   at FindCoin.Block.FindBlock.startTask() in C:\Users\tlan\NEO\FindCoin\FindCoin\Block\FindBlock.cs:line 29
   at FindCoin.core.ContractTask.Start() in C:\Users\tlan\NEO\FindCoin\FindCoin\core\ContractTask.cs:line 61

2018/10/4 14:59:51 [block3] errinfo:
2018/10/4 14:59:51 [block3] System.Net.Http.HttpRequestException
2018/10/4 14:59:51 [block3] No connection could be made because the target machine actively refused it
2018/10/4 14:59:51 [block3]    at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.CreateConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.WaitForCreatedConnectionAsync(ValueTask`1 creationTask)
   at System.Threading.Tasks.ValueTask`1.get_Result()
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at System.Net.HttpWebRequest.SendRequest()
   at System.Net.HttpWebRequest.GetResponse()

2018/10/4 14:59:51 [block3] errinfo:
2018/10/4 14:59:51 [block3] System.Net.Sockets.SocketException
2018/10/4 14:59:51 [block3] No connection could be made because the target machine actively refused it
2018/10/4 14:59:51 [block3]    at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
